src/harness/security_testcase.py

```diff
@@ -266,6 +266,7 @@ def assertTlsHandshakeOrHttpsFailure(self, client_cert=None, client_key=None):
         response = self._sas.Registration(request, ssl_cert=client_cert,
                                           ssl_key=client_key)['registrationResponse']
       except AssertionError as e:
+        logging.debug('HTTPS forbidden: 403')
```

Comment:
This logs what you *hope* has happened, but not necessarily what did happen. Maybe something like "TLS session established, expecting HTTP error 403; received HTTP error %d"?

Reply:
Yes you have right. thanks